airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/util/OssCloudEnvVarConsts.java

```diff
@@ -0,0 +1,8 @@
+package io.airbyte.integrations.util;
+
+public class OssCloudEnvVarConsts {
```

Comment:
let's actually define these in destination-snowflake, since they're specifically what Snowflake is looking for

Reply:
Moved to Destination Snowflake connector. Thanks